server/src/database/migrations/sql/17_add_board_templates.up.sql

```diff
@@ -0,0 +1,21 @@
+create table board_templates
+(
+    id                        uuid              default gen_random_uuid() not null primary key,
+    creator                   uuid              not null references users ON DELETE CASCADE,
+    created_at                timestamptz       default now(),
+    "name"                    varchar(128),
+    description               varchar(300),
+    access_policy             access_policy     not null DEFAULT 'PUBLIC'
+);
+
```

Comment:
templates can be put as favourites, so this needs another column

Reply:
added